dispatch/static/manager/src/styles/components/_dashboard.scss

```diff
@@ -0,0 +1,61 @@
+.dashboard_container {
```

Comment:
Check out the SASS style guide here: https://github.com/ubyssey/ubyssey-dispatch-theme/wiki/SASS-Style-Guide

Sorry, should have let you know before. Shouldn't be hard to change these styles though